src/components/ChatEntry.js

```diff
@@ -1,22 +1,32 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
 
 const ChatEntry = (props) => {
+  const yearsSince = ( 2023 - Number(props.timeStamp.slice(0,4)) )
+  const likeDisplay = props.liked ? '❤️' : '🤍';
+  const localRemote = (props.sender === 'Vladimir') ? 'chat-entry local' : 'chat-entry remote';
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
-      <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+    <div className={localRemote}>
+      <h2 className='entry-name'>{props.sender}</h2>
+      <section className='entry-bubble'>
+        <p>{props.body}</p>
+        <p className='entry-time'>{yearsSince} years ago</p>
+        <button className='like' onClick={() => props.updateLikeStatus(props.id)}>{likeDisplay}</button>
```

Comment:
This onClick is so powerful! Great job lifting up state to ChatLog and eventually App component.